disnake/webhook/async_.py

```diff
@@ -1719,12 +1723,15 @@ async def edit_message(
             .. versionadded:: 2.0
 
         attachments: List[:class:`Attachment`]
-            A list of attachments to keep in the message. If ``[]`` is passed
-            then all existing attachments are removed.
+            A list of attachments to keep in the message.
+            If ``[]`` or ``None`` is passed then all existing attachments are removed.
```

Comment:
parameter annotation not updated

Reply:
https://github.com/DisnakeDev/disnake/pull/457/commits/3dc30b1af1fdbfd9fddbd3ab07370b36cae42029